docs/capacity.md

```diff
@@ -47,6 +47,9 @@ As an example:
 So a system with 4GB of memory would be capable of running 20 forks. The value `mem_per_fork` can be controlled by setting the AWX settings value
 (or environment variable) `SYSTEM_TASK_FORKS_MEM` which defaults to `100`.
 
+The value "mem" can be controlled by configuring the SYSTEM_TASK_ABS_MEM settings.
+
+The configurations can be specified in settings in awx-config-maps of AWX deployment or be set as environment variable in the AWX deployment of AWX Kubernetes version.
```

Comment:
I think we need to reword this sentence, maybe elaborate a little more on how to add this setting? you can point to the section in the awx-operator readme on how to add extra settings